apps/els_dap/erlang_ls.config

```diff
@@ -0,0 +1,2 @@
+include_dirs:
```

Comment:
Do we really need per-app configs?

Reply:
The LS will not find includes otherwise, at least not for VSCode.

My hunch is, that the LS does not understand the Umbrella structure well.

Reply:
I use a config something like this for rebar3 umbrella projects (some of these might be default for erlang_ls already)
```
apps_dirs:
  - "apps/*"
deps_dirs:
  - "_build/*/lib/*"
include_dirs:
 # for include
  - "include"
  - "apps/*/include"
  - "_build/*/lib/*/include"
# for include_lib
  - "apps"
  - "_build/*/lib/"
```

Reply:
There are only defaults set for the base-case of one application:
```
  Paths = [ els_utils:resolve_paths( [ [RootPath, Dir, "src"]
                                     , [RootPath, Dir, "test"]
                                     , [RootPath, Dir, "include"]
                                     ]
                                   , RootPath
                                   , Recursive
                                   )
```
And there is an lsp instance started for each project app in vscode (plus one for the umbrella app)

Reply:
alright, having a central config for the umbrella works with:
```
apps_dirs:
    - "apps/*"
deps_dirs:
  - "_build/default/lib/*"
include_dirs:
  - "apps"
  - "apps/*/include"
  - "_build/default/lib/"
  ```
  
  It's not entirely correct, as the difference between `-include_lib` and `-include` are not picked up by the LS